build.sbt

```diff
@@ -416,15 +415,13 @@ lazy val compiler = configureAsSubproject(project)
     mappings in Compile in packageSrc ++= {
       val base = (unmanagedResourceDirectories in Compile).value ++
         (unmanagedResourceDirectories in Compile in LocalProject("interactive")).value ++
-        (unmanagedResourceDirectories in Compile in LocalProject("scaladoc")).value ++
         (unmanagedResourceDirectories in Compile in LocalProject("repl")).value
       base ** ((includeFilter in unmanagedResources in Compile).value || "*.scala" || "*.psd" || "*.ai" || "*.java") pair relativeTo(base)
     },
-    // Include the additional projects in the scaladoc JAR:
+    // Include the additional projects in the compiler JAR:
```

Comment:
this was correct the way it was btw, it means the jar of scaladocs of the compiler